src/Arcus.Scripting.ARM/Scripts/Inject-ArmContent.ps1

```diff
@@ -0,0 +1,113 @@
+<#
+    Possible injection instructions in ARM templates or recursively referenced files:
+    
+    ${ fileToInject.xml }
+    ${ FileToInject=file.xml }
+    ${ FileToInject = ".\Parent Directory\file.xml" }
+    ${ FileToInject = ".\Parent Directory\file.xml", EscapeJson, ReplaceSpecialChars }
+    ${ FileToInject = '.\Parent Directory\file.json', InjectAsJsonObject }
+ #>
+
+param (
+    [string] $Path = $PSScriptRoot
+)
+
+function InjectFile {
+    param(
+        [string] $filePath
+    )
+
+    Write-Host "Checking file $filePath" 
+
+    $replaceContentDelegate = {
```

Comment:
Can't we use a func instead of a delegate?

Reply:
This was just moved as-is, maybe something to do with the expected parameter type.

Reply:
Ok no problem then!